README.md

```diff
@@ -65,7 +65,8 @@ pipenv install
     - If you see `Python 3.8.0` or higher, skip to step 4
     - If you see `Python 3.7.9` or lower, skip to step 2
     - If you see `python : The term 'python' is not recognized`, continue to step 2
-2. Install the latest [Python 3.8](https://www.microsoft.com/en-us/p/python-38/9mssztt1n39l) for Windows
+2. Install the latest [Python 3.8](https://www.python.org/ftp/python/3.8.6/python-3.8.6-amd64.exe) for Windows
+    - Click `Add Path` at the bottom of the first installation page.
```

Comment:
Simplifies the installation path of python so that the user does not have to do step #3

Reply:
Step 3 shouldn't be necessary anyway, it's more about cleaning up old Python stuff on the path. The Windows Store installation adds itself to the PATH automatically without having to check a box.

Reply:
I definitely agree.  I considered editing this to remove it entirely, but it seemed a bit of a shame to remove all that hard work that someone typed out in 3 so the changes I made were to try and accommodate keeping it while still informing users of how to go a simpler route in the first place